src/SeleniumLibrary/keywords/element.py

```diff
@@ -335,6 +335,36 @@ def element_text_should_be(self, locator, expected, message=None, ignore_case=Fa
                            % (locator, expected, before_text))
             raise AssertionError(message)
 
+    @keyword
+    def element_text_should_not_be(self, locator, not_expected, message=None, ignore_case=False):
+        """Verifies that element ``locator`` contains exact text ``expected``.
```

Comment:
The doc string contains few copy paste errors. Please also see also line 351 and 353.